src/test/java/com/google/javascript/gents/singleTests/goog_scope.js

```diff
@@ -0,0 +1,27 @@
+goog.provide("A.B");
+goog.provide("A.B.C");
+
+goog.scope(function() {
+
```

Comment:
a common pattern is to alias "require" results here (`var LocalName = goog.something.LocalName;`). Maybe add a test case?